_data/instagram.yml

```diff
@@ -4,6 +4,6 @@ blog:
   link: /blog/tech-stories/5-most-valuable-things-i-learned-in-my-10-plus-year-career/
 
 job:
-  image: /assets/img/pages/headers/public/journey-developer.jpg
-  text: We're hiring a<br /><strong>Ruby Developer (m/f/d)</strong>
-  link: https://epages-gmbh.jobs.personio.de/job/202168?language=en
+  image: /assets/img/pages/headers/public/onboarding.jpg
+  text: We're hiring a<br /><strong style="font-size:2.0rem;">Software Developer - 3rd Party Integration, REST API (m/f/d)</strong>
```

Comment:
Did you get the "style" part from Team 42 or have it reviewed by them?

Reply:
No, but I've tested it myself. Due to the long title, this font size looks best.

Reply:
If you want to I can show you the result tomorrow.

Reply:
If you've tested it and you're sure that it works, that is fine for me :)